pyproject.toml

```diff
@@ -16,14 +16,14 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
-
 dependencies = [
-    "git+https://github.com/OpenPecha/fast-antx.git",
+  "pandas<=2.2.3"
```

Comment:
why did u remove that dependency. we need that dependency later for transfer

Reply:
I didn’t remove anything manually. After pulling the code, the changes to the dependencies didn’t reflect in my local setup. When I added the following dependency:
"git+https://github.com/OpenPecha/fast-antx.git"
It resulted in an error, stating that pyproject.toml does not allow Git-based dependencies to be specified in this format. To fix the issue, I changed it to:
"fast-antx @ git+https://github.com/OpenPecha/fast-antx.git"